darts/timeseries.py

```diff
@@ -1499,123 +1840,169 @@ def __rtruediv__(self, n):
         return n * (self ** (-1))
 
     def __abs__(self):
-        series = abs(self._df)
-        return TimeSeries(series, self.freq_str())
+        return TimeSeries(abs(self._xa))
```

Comment:
Do you know how all of that is doing performance wise. We convert a lot from and to xarray and TimeSeries and this is probably sometimes done on large time series (maybe 10k samples or so?).

Reply:
This is not super efficient, because it requires (i) creating a new object (object creation is expensive in itself), and (ii) each TimeSeries object created copies its underlying DataArray, which for sure adds an extra cost. However I'm not sure how to go around this (and keep all our nice properties)